echopype/tests/test_process_ad2cp.py

```diff
@@ -102,10 +102,7 @@ def test_field(base, test, base_field_name, test_field_name):
         base = xr.open_dataset(parsed_file, group="Data/Avg")
         
         test = xr.open_dataset(test_file, group="Platform")
-        # does not work because ocean contour timestamps have rounding errors
-        # so we can't compare times
-        # for time in base["time"]:
-        #     assert close(base["Heading"].sel(time=time).data[()], test["heading"].sel(time=time).data[()])
+        # ocean contour timestamps have rounding errors so we can't compare times
```

Comment:
Could you plot a histogram of the timestamp difference and we can discuss in our next meeting? Is it on the order of ns or ms?